### ISSUE TITLE
False positive on Intl.Locale

### ISSUE THREAD

**Issue by mrmckeb** [3ru/eslint-plugin-baseline-js#102]
This is widely available, and has been from 2020.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale

```ts
const x = new Locale('en-US').language;
// Feature 'Intl.Locale info' (intl-locale-info) has Limited availability and exceeds 2024.eslint(baseline-js/use-baseline)
```

**Comment by mrmckeb** [comment]
Thank you!
